Sources/GRPC/ConnectionPool/GRPCChannelPool.swift

```diff
@@ -191,6 +195,10 @@ extension GRPCChannelPool {
         return SwiftLogNoOpLogHandler()
       }
     )
+
+    #if canImport(Network)
+    public var transportServices: TransportServices = .defaults
```

Comment:
Probably worth adding a comment saying that this will be ignored unless an appropriate event loop group is used.